### DISCUSSION TITLE
How to show SVG images in Net 8.0 XAML?

### DISCUSSION THREAD

**Discussion by DKGH** [wieslawsoltes/Svg.Skia#226] [Q&A]
I am having difficulty following the main documentation in simply showing the contents of an SVG file. Do I need to install another nugget package? Please share step by step changes needed in XAML and code behind. I was using an <Image /> to show png. I want to replace it to show SVG file.

Adding more details:

Usually I add the control (like <Image>) in XAML and then add some code behind to update it as needed. However, I am unable to figure out how to do that here.

1. From README.md:
```
SKCanvas canvas = ...
canvas.DrawPicture(svg.Picture);
```
What do I need to put at `...`?

2. This isn't compatible with a NET 8 WPF application: <Svg Path="/Assets/__AJ_Digital_Camera.svg"/>
3. WPF image can't source from SvgImage: <Image Source="{SvgImage /Assets/__AJ_Digital_Camera.svg}"/>
4. From the provided samples, one project compiled for me (using VS 2022). Out of that a mention of SKCanvasControl looks promising. Examining this further.

**Comment by wieslawsoltes** [comment]
What xaml?

**Reply by DKGH** [reply]
User control like XAML?
